src/apix-components/swagger-console.html

```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 <!--
-  ~ Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+  ~ Copyright (c) 2017 VMware, Inc. All Rights Reserved.
```

Comment:
strictly speaking this should be changed to be 2016-2017.  You never remove the first copyright year, you just change the last one to indicate the first year that it was introduced.  That applies to all code that is changed year to year.

Reply:
fixed all files